### PR TITLE
feat(): bump to latest

### PR DESCRIPTION
**Pull request by mhartington**

### CODE REVIEW

File: angular/base/angular.json [hunk 1]
```diff
@@ -25,11 +25,6 @@
                 "glob": "**/*",
                 "input": "src/assets",
                 "output": "assets"
-              },
-              {
-                "glob": "**/*.svg",
-                "input": "node_modules/@ionic/angular/dist/ionic/svg",
-                "output": "./svg"
```
Comment (dwieeb) [inline angular/base/angular.json]: 👏👏👏

File: angular/base/package.json [hunk 2]
```diff
@@ -13,45 +13,45 @@
...
+    "@angular-devkit/architect": "~0.10.0",
+    "@angular-devkit/build-angular": "~0.10.0",
+    "@angular-devkit/core": "~7.0.0",
+    "@angular-devkit/schematics": "~7.0.0",
+    "@angular/cli": "~7.0.0",
+    "@angular/compiler": "~7.0.0",
+    "@angular/compiler-cli": "~7.0.0",
+    "@angular/language-service": "~7.0.0",
+    "@ionic/angular-toolkit": "~1.2.0",
+    "@types/node": "~8.9.4",
```
Comment (dwieeb) [inline angular/base/package.json]: Let's keep node 10

File: angular/base/src/tsconfig.app.json [hunk 3]
```diff
@@ -2,8 +2,7 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "outDir": "../out-tsc/app",
-    "baseUrl": "./",
-    "module": "es2015"
+    "types": []
```
Comment (dwieeb) [inline angular/base/src/tsconfig.app.json]: This is not ideal, but it looks like Angular starters do this, so ¯\_(ツ)_/¯

Comment (mhartington) [reply angular/base/src/tsconfig.app.json]: Yeah, Im not sure on the side effects of this, but if it's done the same way in a base angular project, we should match it

**Pull request merged**